libraries/wallet/wallet.cpp

```diff
@@ -2997,7 +3043,7 @@ vector<operation_detail> wallet_api::get_account_history(string name, int limit)
 
       int page_limit = skip_first_row ? std::min( 100, limit + 1 ) : std::min( 100, limit );
 
-      vector<operation_history_object> current = my->_remote_hist->get_account_history( name, operation_history_id_type(),
+      vector<operation_history_object> current = my->_remote_hist->get_account_history( always_id, operation_history_id_type(),
```

Comment:
It seems Github is limiting line lengths to 123 characters without scrolling. Please adjust ( it is currently cut off at operation_history_id_ty )